2018-09-14 01:05:09.524 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:05:09.528 30120 DEBUG cinder.manager [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-14 01:05:09.594 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:05:09.596 30120 DEBUG cinder.volume.drivers.lvm [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-14 01:05:09.598 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:05:11.261 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.663s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:05:11.265 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:05:12.578 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.314s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:05:12.580 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:05:13.603 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.023s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:05:13.606 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:05:14.508 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.902s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:06:09.522 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:06:09.524 30120 DEBUG cinder.manager [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-14 01:06:09.603 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:06:09.605 30120 DEBUG cinder.volume.drivers.lvm [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-14 01:06:09.608 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:06:11.021 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.413s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:06:11.025 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:06:12.250 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.225s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:06:12.257 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:06:13.290 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.035s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:06:13.293 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:06:14.359 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.067s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:07:09.526 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:07:09.530 30120 DEBUG cinder.manager [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-14 01:07:09.602 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:07:09.604 30120 DEBUG cinder.volume.drivers.lvm [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-14 01:07:09.606 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:07:10.871 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.266s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:07:10.875 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:07:12.072 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.197s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:07:12.077 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:07:13.237 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.161s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:07:13.240 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:07:14.453 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.214s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:08:09.530 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:08:09.535 30120 DEBUG cinder.manager [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-14 01:08:09.623 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:08:09.624 30120 DEBUG cinder.volume.drivers.lvm [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-14 01:08:09.627 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:08:10.691 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.065s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:08:10.694 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:08:11.641 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.947s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:08:11.644 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:08:12.906 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.262s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:08:12.909 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:08:13.895 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.986s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:09:09.532 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:09:09.536 30120 DEBUG cinder.manager [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-14 01:09:09.627 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:09:09.628 30120 DEBUG cinder.volume.drivers.lvm [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-14 01:09:09.630 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:09:10.828 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.198s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:09:10.832 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:09:11.893 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.062s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:09:11.898 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:09:12.912 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.015s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:09:12.916 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:09:14.091 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.176s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:10:09.535 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:10:09.538 30120 DEBUG cinder.manager [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-14 01:10:09.591 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:10:09.592 30120 DEBUG cinder.volume.drivers.lvm [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-14 01:10:09.594 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:10:11.018 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.424s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:10:11.021 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:10:12.513 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.493s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:10:12.516 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:10:13.705 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.190s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:10:13.709 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:10:15.143 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.435s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:10:57.783 30120 DEBUG cinder.volume.flows.manager.create_volume [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Volume reschedule parameters: True retry: {u'num_attempts': 1, u'backends': [u'localhost.localdomain@lvm#lvm'], u'hosts': [u'localhost.localdomain@lvm#lvm']} get_flow /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:1126
2018-09-14 01:10:57.845 30120 DEBUG cinder.volume.manager [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Flow 'volume_create_manager' (ae1413d3-c7f5-4fb7-b597-30d4f5bcfbc3) transitioned into state 'RUNNING' from state 'PENDING' _flow_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:145
2018-09-14 01:10:57.860 30120 DEBUG cinder.volume.manager [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Task 'cinder.volume.flows.manager.create_volume.ExtractVolumeRefTask;volume:create' (dfc8e073-9c8d-4f5d-a14b-054fc2b9847d) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-14 01:10:57.921 30120 DEBUG oslo_db.sqlalchemy.engines [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-09-14 01:10:57.985 30120 DEBUG cinder.volume.manager [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Task 'cinder.volume.flows.manager.create_volume.ExtractVolumeRefTask;volume:create' (dfc8e073-9c8d-4f5d-a14b-054fc2b9847d) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'Volume(_name_id=None,admin_metadata={},attach_status='detached',availability_zone='nova',bootable=False,cluster=<?>,cluster_name=None,consistencygroup=<?>,consistencygroup_id=None,created_at=2018-09-14T05:10:57Z,deleted=False,deleted_at=None,display_description=None,display_name='tempest-VOLUME_SAMPLE-1536901511',ec2_id=None,encryption_key_id=None,glance_metadata=<?>,group=<?>,group_id=None,host='localhost.localdomain@lvm#lvm',id=181eb2b7-599c-48f1-b589-bf248ab2da71,launched_at=None,metadata={},migration_status=None,multiattach=False,previous_status=None,project_id='f4b8aa14ab144c71ad8c02aaabb0b4f4',provider_auth=None,provider_geometry=None,provider_id=None,provider_location=None,replication_driver_data=None,replication_extended_status=None,replication_status=None,scheduled_at=2018-09-14T05:10:57Z,size=1,snapshot_id=None,snapshots=<?>,source_volid=None,status='creating',terminated_at=None,updated_at=2018-09-14T05:10:57Z,user_id='923cc1ab932046e9bf77dcd7ee5191ad',volume_attachment=<?>,volume_type=VolumeType(d9c60c05-9188-4fa8-885a-e4f8424aa313),volume_type_id=d9c60c05-9188-4fa8-885a-e4f8424aa313)' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-14 01:10:57.993 30120 DEBUG cinder.volume.manager [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Task 'cinder.volume.flows.manager.create_volume.OnFailureRescheduleTask;volume:create' (43bf7982-11c7-4ab0-afdf-f94d5b2fd38f) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-14 01:10:57.998 30120 DEBUG cinder.volume.manager [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Task 'cinder.volume.flows.manager.create_volume.OnFailureRescheduleTask;volume:create' (43bf7982-11c7-4ab0-afdf-f94d5b2fd38f) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'None' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-14 01:10:58.004 30120 DEBUG cinder.volume.manager [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Task 'cinder.volume.flows.manager.create_volume.ExtractVolumeSpecTask;volume:create' (85d96651-0624-4c6e-ab30-26369a0fd37c) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-14 01:10:58.009 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f3d9e91bb792acb4a4af63c3b6a0256332f24d79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7' http://10.0.20.25:9292/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618 log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-14 01:10:58.731 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] GET call to image for http://10.0.20.25:9292/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618 used request id req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-14 01:10:58.733 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] 
HTTP/1.1 200 OK
Content-Length: 616
Content-Type: application/json
X-Openstack-Request-Id: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7
Date: Fri, 14 Sep 2018 05:10:58 GMT
Connection: keep-alive

{"status": "active", "name": "tempest-cirros-0.4.0-x86_64-1536901511", "tags": [], "container_format": "bare", "created_at": "2018-09-14T05:06:10Z", "size": 12716032, "disk_format": "qcow2", "updated_at": "2018-09-14T05:06:11Z", "visibility": "public", "self": "/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618", "min_disk": 0, "protected": false, "id": "0bf12814-cbac-4f29-8b63-7c96c2f8b618", "file": "/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618/file", "checksum": "443b7623e27ecf03dc9e01ee93f67afe", "owner": "bd8a53962fd64ffeab7f2932b0e688c5", "virtual_size": null, "min_ram": 0, "schema": "/v2/schemas/image"}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-14 01:10:58.735 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f3d9e91bb792acb4a4af63c3b6a0256332f24d79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7' http://10.0.20.25:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-14 01:10:58.764 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] GET call to image for http://10.0.20.25:9292/v2/schemas/image used request id req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-14 01:10:58.768 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4157
X-Openstack-Request-Id: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7
Date: Fri, 14 Sep 2018 05:10:58 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Status of the image"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "List of strings related to the image"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Format of the container"}, "min_ram": {"type": "integer", "description": "Amount of ram (in MB) required to boot image."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "A set of URLs to access the image file kept in external store"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Scope of image accessibility"}, "updated_at": {"readOnly": true, "type": "string", "description": "Date and time of the last image modification"}, "owner": {"type": ["null", "string"], "description": "Owner of the image", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "An image file url"}, "min_disk": {"type": "integer", "description": "Amount of disk space (in GB) required to boot image."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Virtual size of image in bytes"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "An identifier for the image"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Size of image file in bytes"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Descriptive name for the image", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "md5 hash of image contents.", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Date and time of image registration"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Format of the disk"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "If true, image will not be deletable."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL to access the image file kept in external store"}, "self": {"readOnly": true, "type": "string", "description": "An image self url"}, "schema": {"readOnly": true, "type": "string", "description": "An image schema url"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-14 01:10:58.773 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f3d9e91bb792acb4a4af63c3b6a0256332f24d79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7' http://10.0.20.25:9292/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618 log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-14 01:10:58.831 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] GET call to image for http://10.0.20.25:9292/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618 used request id req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-14 01:10:58.833 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] 
HTTP/1.1 200 OK
Content-Length: 616
Content-Type: application/json
X-Openstack-Request-Id: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7
Date: Fri, 14 Sep 2018 05:10:58 GMT
Connection: keep-alive

{"status": "active", "name": "tempest-cirros-0.4.0-x86_64-1536901511", "tags": [], "container_format": "bare", "created_at": "2018-09-14T05:06:10Z", "size": 12716032, "disk_format": "qcow2", "updated_at": "2018-09-14T05:06:11Z", "visibility": "public", "self": "/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618", "min_disk": 0, "protected": false, "id": "0bf12814-cbac-4f29-8b63-7c96c2f8b618", "file": "/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618/file", "checksum": "443b7623e27ecf03dc9e01ee93f67afe", "owner": "bd8a53962fd64ffeab7f2932b0e688c5", "virtual_size": null, "min_ram": 0, "schema": "/v2/schemas/image"}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-14 01:10:58.835 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f3d9e91bb792acb4a4af63c3b6a0256332f24d79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7' http://10.0.20.25:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-14 01:10:58.855 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] GET call to image for http://10.0.20.25:9292/v2/schemas/image used request id req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-14 01:10:58.861 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4157
X-Openstack-Request-Id: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7
Date: Fri, 14 Sep 2018 05:10:58 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Status of the image"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "List of strings related to the image"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Format of the container"}, "min_ram": {"type": "integer", "description": "Amount of ram (in MB) required to boot image."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "A set of URLs to access the image file kept in external store"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Scope of image accessibility"}, "updated_at": {"readOnly": true, "type": "string", "description": "Date and time of the last image modification"}, "owner": {"type": ["null", "string"], "description": "Owner of the image", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "An image file url"}, "min_disk": {"type": "integer", "description": "Amount of disk space (in GB) required to boot image."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Virtual size of image in bytes"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "An identifier for the image"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Size of image file in bytes"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Descriptive name for the image", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "md5 hash of image contents.", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Date and time of image registration"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Format of the disk"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "If true, image will not be deletable."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL to access the image file kept in external store"}, "self": {"readOnly": true, "type": "string", "description": "An image self url"}, "schema": {"readOnly": true, "type": "string", "description": "An image schema url"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-14 01:10:58.866 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f3d9e91bb792acb4a4af63c3b6a0256332f24d79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7' http://10.0.20.25:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-14 01:10:58.890 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] GET call to image for http://10.0.20.25:9292/v2/schemas/image used request id req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-14 01:10:58.892 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4157
X-Openstack-Request-Id: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7
Date: Fri, 14 Sep 2018 05:10:58 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Status of the image"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "List of strings related to the image"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Format of the container"}, "min_ram": {"type": "integer", "description": "Amount of ram (in MB) required to boot image."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "A set of URLs to access the image file kept in external store"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Scope of image accessibility"}, "updated_at": {"readOnly": true, "type": "string", "description": "Date and time of the last image modification"}, "owner": {"type": ["null", "string"], "description": "Owner of the image", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "An image file url"}, "min_disk": {"type": "integer", "description": "Amount of disk space (in GB) required to boot image."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Virtual size of image in bytes"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "An identifier for the image"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Size of image file in bytes"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Descriptive name for the image", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "md5 hash of image contents.", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Date and time of image registration"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Format of the disk"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "If true, image will not be deletable."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL to access the image file kept in external store"}, "self": {"readOnly": true, "type": "string", "description": "An image self url"}, "schema": {"readOnly": true, "type": "string", "description": "An image schema url"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-14 01:10:58.899 30120 DEBUG cinder.volume.manager [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Task 'cinder.volume.flows.manager.create_volume.ExtractVolumeSpecTask;volume:create' (85d96651-0624-4c6e-ab30-26369a0fd37c) transitioned into state 'SUCCESS' from state 'RUNNING' with result '{'status': u'creating', 'image_location': (None, None), 'image_id': u'0bf12814-cbac-4f29-8b63-7c96c2f8b618', 'volume_id': u'181eb2b7-599c-48f1-b589-bf248ab2da71', 'volume_size': 1, 'image_service': <cinder.image.glance.GlanceImageService object at 0x7fd475d09ad0>, 'image_meta': {u'status': u'active', u'name': u'tempest-cirros-0.4.0-x86_64-1536901511', u'tags': [], u'container_format': u'bare', u'created_at': datetime.datetime(2018, 9, 14, 5, 6, 10, tzinfo=<iso8601.Utc>), u'disk_format': u'qcow2', u'updated_at': datetime.datetime(2018, 9, 14, 5, 6, 11, tzinfo=<iso8601.Utc>), u'visibility': u'public', 'properties': {}, u'owner': u'bd8a53962fd64ffeab7f2932b0e688c5', u'protected': False, u'id': u'0bf12814-cbac-4f29-8b63-7c96c2f8b618', u'file': u'/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618/file', u'checksum': u'443b7623e27ecf03dc9e01ee93f67afe', u'min_disk': 0, u'virtual_size': None, u'min_ram': 0, u'size': 12716032}, 'volume_name': u'volume-181eb2b7-599c-48f1-b589-bf248ab2da71', 'type': 'image'}' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-14 01:10:58.905 30120 DEBUG cinder.volume.manager [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Task 'cinder.volume.flows.manager.create_volume.NotifyVolumeActionTask;volume:create, create.start' (f0429595-2eea-43fc-9ee3-5e3d4bf1a7c4) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-14 01:10:58.967 30120 DEBUG cinder.volume.manager [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Task 'cinder.volume.flows.manager.create_volume.NotifyVolumeActionTask;volume:create, create.start' (f0429595-2eea-43fc-9ee3-5e3d4bf1a7c4) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'None' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-14 01:10:58.974 30120 DEBUG cinder.volume.manager [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Task 'cinder.volume.flows.manager.create_volume.CreateVolumeFromSpecTask;volume:create' (c7df6b34-7675-45a8-8653-61bf26c5f6e0) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-14 01:10:58.978 30120 INFO cinder.volume.flows.manager.create_volume [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Volume 181eb2b7-599c-48f1-b589-bf248ab2da71: being created as image with specification: {'status': u'creating', 'image_location': (None, None), 'volume_size': 1, 'volume_name': u'volume-181eb2b7-599c-48f1-b589-bf248ab2da71', 'image_id': u'0bf12814-cbac-4f29-8b63-7c96c2f8b618', 'image_service': <cinder.image.glance.GlanceImageService object at 0x7fd475d09ad0>, 'image_meta': {u'status': u'active', u'name': u'tempest-cirros-0.4.0-x86_64-1536901511', u'tags': [], u'container_format': u'bare', u'created_at': datetime.datetime(2018, 9, 14, 5, 6, 10, tzinfo=<iso8601.Utc>), u'disk_format': u'qcow2', u'updated_at': datetime.datetime(2018, 9, 14, 5, 6, 11, tzinfo=<iso8601.Utc>), u'visibility': u'public', 'properties': {}, u'owner': u'bd8a53962fd64ffeab7f2932b0e688c5', u'protected': False, u'id': u'0bf12814-cbac-4f29-8b63-7c96c2f8b618', u'file': u'/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618/file', u'checksum': u'443b7623e27ecf03dc9e01ee93f67afe', u'min_disk': 0, u'virtual_size': None, u'min_ram': 0, u'size': 12716032}}
2018-09-14 01:10:58.979 30120 DEBUG cinder.volume.flows.manager.create_volume [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Cloning 181eb2b7-599c-48f1-b589-bf248ab2da71 from image 0bf12814-cbac-4f29-8b63-7c96c2f8b618  at location (None, None). _create_from_image /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:896
2018-09-14 01:10:58.985 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f3d9e91bb792acb4a4af63c3b6a0256332f24d79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7' http://10.0.20.25:9292/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618/file log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-14 01:10:59.063 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] GET call to image for http://10.0.20.25:9292/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618/file used request id req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-14 01:10:59.065 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] 
HTTP/1.1 200 OK
Content-Type: application/octet-stream
Content-Md5: 443b7623e27ecf03dc9e01ee93f67afe
Content-Length: 12716032
X-Openstack-Request-Id: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7
Date: Fri, 14 Sep 2018 05:10:59 GMT
Connection: keep-alive
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-14 01:10:59.477 30120 DEBUG cinder.image.image_utils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Image fetch details: dest /var/lib/cinder/conversion/tmpY4uHZXlocalhost.localdomain@lvm, size 12.00 MB, duration 1.00 sec fetch /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:247
2018-09-14 01:10:59.478 30120 INFO cinder.image.image_utils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Image download 12.00 MB at 12.00 MB/s
2018-09-14 01:10:59.481 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f3d9e91bb792acb4a4af63c3b6a0256332f24d79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7' http://10.0.20.25:9292/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618 log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-14 01:10:59.567 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] GET call to image for http://10.0.20.25:9292/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618 used request id req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-14 01:10:59.569 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] 
HTTP/1.1 200 OK
Content-Length: 616
Content-Type: application/json
X-Openstack-Request-Id: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7
Date: Fri, 14 Sep 2018 05:10:59 GMT
Connection: keep-alive

{"status": "active", "name": "tempest-cirros-0.4.0-x86_64-1536901511", "tags": [], "container_format": "bare", "created_at": "2018-09-14T05:06:10Z", "size": 12716032, "disk_format": "qcow2", "updated_at": "2018-09-14T05:06:11Z", "visibility": "public", "self": "/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618", "min_disk": 0, "protected": false, "id": "0bf12814-cbac-4f29-8b63-7c96c2f8b618", "file": "/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618/file", "checksum": "443b7623e27ecf03dc9e01ee93f67afe", "owner": "bd8a53962fd64ffeab7f2932b0e688c5", "virtual_size": null, "min_ram": 0, "schema": "/v2/schemas/image"}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-14 01:10:59.571 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f3d9e91bb792acb4a4af63c3b6a0256332f24d79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7' http://10.0.20.25:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-14 01:10:59.600 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] GET call to image for http://10.0.20.25:9292/v2/schemas/image used request id req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-14 01:10:59.604 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4157
X-Openstack-Request-Id: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7
Date: Fri, 14 Sep 2018 05:10:59 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Status of the image"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "List of strings related to the image"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Format of the container"}, "min_ram": {"type": "integer", "description": "Amount of ram (in MB) required to boot image."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "A set of URLs to access the image file kept in external store"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Scope of image accessibility"}, "updated_at": {"readOnly": true, "type": "string", "description": "Date and time of the last image modification"}, "owner": {"type": ["null", "string"], "description": "Owner of the image", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "An image file url"}, "min_disk": {"type": "integer", "description": "Amount of disk space (in GB) required to boot image."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Virtual size of image in bytes"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "An identifier for the image"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Size of image file in bytes"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Descriptive name for the image", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "md5 hash of image contents.", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Date and time of image registration"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Format of the disk"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "If true, image will not be deletable."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL to access the image file kept in external store"}, "self": {"readOnly": true, "type": "string", "description": "An image self url"}, "schema": {"readOnly": true, "type": "string", "description": "An image schema url"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-14 01:10:59.615 30120 DEBUG oslo_concurrency.processutils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpY4uHZXlocalhost.localdomain@lvm execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:11:01.437 30120 DEBUG oslo_concurrency.processutils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpY4uHZXlocalhost.localdomain@lvm" returned: 0 in 1.822s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:11:01.443 30120 DEBUG cinder.image.image_utils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Temporary image 0bf12814-cbac-4f29-8b63-7c96c2f8b618 is fetched for user 923cc1ab932046e9bf77dcd7ee5191ad. fetch /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:672
2018-09-14 01:11:01.446 30120 DEBUG oslo_concurrency.processutils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpY4uHZXlocalhost.localdomain@lvm execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:11:03.012 30120 DEBUG oslo_concurrency.processutils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpY4uHZXlocalhost.localdomain@lvm" returned: 0 in 1.566s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:11:03.016 30120 DEBUG oslo_concurrency.processutils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvcreate -T -V 1g -n volume-181eb2b7-599c-48f1-b589-bf248ab2da71 cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:11:04.455 30120 DEBUG oslo_concurrency.processutils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvcreate -T -V 1g -n volume-181eb2b7-599c-48f1-b589-bf248ab2da71 cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.440s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:11:04.529 30120 DEBUG cinder.volume.flows.manager.create_volume [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Attempting download of 0bf12814-cbac-4f29-8b63-7c96c2f8b618 ((None, None)) to volume 181eb2b7-599c-48f1-b589-bf248ab2da71. _copy_image_to_volume /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:536
2018-09-14 01:11:04.533 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f3d9e91bb792acb4a4af63c3b6a0256332f24d79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7' http://10.0.20.25:9292/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618 log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-14 01:11:04.592 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] GET call to image for http://10.0.20.25:9292/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618 used request id req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-14 01:11:04.598 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] 
HTTP/1.1 200 OK
Content-Length: 616
Content-Type: application/json
X-Openstack-Request-Id: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7
Date: Fri, 14 Sep 2018 05:11:04 GMT
Connection: keep-alive

{"status": "active", "name": "tempest-cirros-0.4.0-x86_64-1536901511", "tags": [], "container_format": "bare", "created_at": "2018-09-14T05:06:10Z", "size": 12716032, "disk_format": "qcow2", "updated_at": "2018-09-14T05:06:11Z", "visibility": "public", "self": "/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618", "min_disk": 0, "protected": false, "id": "0bf12814-cbac-4f29-8b63-7c96c2f8b618", "file": "/v2/images/0bf12814-cbac-4f29-8b63-7c96c2f8b618/file", "checksum": "443b7623e27ecf03dc9e01ee93f67afe", "owner": "bd8a53962fd64ffeab7f2932b0e688c5", "virtual_size": null, "min_ram": 0, "schema": "/v2/schemas/image"}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-14 01:11:04.601 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f3d9e91bb792acb4a4af63c3b6a0256332f24d79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7' http://10.0.20.25:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-14 01:11:04.629 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] GET call to image for http://10.0.20.25:9292/v2/schemas/image used request id req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-14 01:11:04.631 30120 DEBUG glanceclient.common.http [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4157
X-Openstack-Request-Id: req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7
Date: Fri, 14 Sep 2018 05:11:04 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Status of the image"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "List of strings related to the image"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Format of the container"}, "min_ram": {"type": "integer", "description": "Amount of ram (in MB) required to boot image."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "A set of URLs to access the image file kept in external store"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Scope of image accessibility"}, "updated_at": {"readOnly": true, "type": "string", "description": "Date and time of the last image modification"}, "owner": {"type": ["null", "string"], "description": "Owner of the image", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "An image file url"}, "min_disk": {"type": "integer", "description": "Amount of disk space (in GB) required to boot image."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Virtual size of image in bytes"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "An identifier for the image"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Size of image file in bytes"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Descriptive name for the image", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "md5 hash of image contents.", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Date and time of image registration"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Format of the disk"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "If true, image will not be deletable."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL to access the image file kept in external store"}, "self": {"readOnly": true, "type": "string", "description": "An image self url"}, "schema": {"readOnly": true, "type": "string", "description": "An image schema url"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-14 01:11:04.644 30120 DEBUG oslo_concurrency.processutils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpFDAKJw execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:11:05.835 30120 DEBUG oslo_concurrency.processutils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpFDAKJw" returned: 0 in 1.192s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:11:05.840 30120 DEBUG oslo_concurrency.processutils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpY4uHZXlocalhost.localdomain@lvm execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:11:07.086 30120 DEBUG oslo_concurrency.processutils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpY4uHZXlocalhost.localdomain@lvm" returned: 0 in 1.247s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:11:07.089 30120 DEBUG cinder.image.image_utils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] 0bf12814-cbac-4f29-8b63-7c96c2f8b618 was qcow2, converting to raw  fetch_to_volume_format /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:417
2018-09-14 01:11:07.091 30120 DEBUG oslo_concurrency.processutils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf dd count=0 if=/var/lib/cinder/conversion/tmpY4uHZXlocalhost.localdomain@lvm of=/dev/mapper/cinder--volumes2-volume--181eb2b7--599c--48f1--b589--bf248ab2da71 oflag=direct execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:11:08.418 30120 DEBUG oslo_concurrency.processutils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf dd count=0 if=/var/lib/cinder/conversion/tmpY4uHZXlocalhost.localdomain@lvm of=/dev/mapper/cinder--volumes2-volume--181eb2b7--599c--48f1--b589--bf248ab2da71 oflag=direct" returned: 0 in 1.327s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:11:08.422 30120 DEBUG oslo_concurrency.processutils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf qemu-img convert -t none -f qcow2 -O raw /var/lib/cinder/conversion/tmpY4uHZXlocalhost.localdomain@lvm /dev/mapper/cinder--volumes2-volume--181eb2b7--599c--48f1--b589--bf248ab2da71 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:11:09.538 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:11:09.539 30120 DEBUG cinder.manager [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-14 01:11:09.593 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:11:09.594 30120 DEBUG cinder.volume.drivers.lvm [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-14 01:11:09.597 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:11:11.010 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.413s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:11:11.025 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:11:11.403 30120 DEBUG oslo_concurrency.processutils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf qemu-img convert -t none -f qcow2 -O raw /var/lib/cinder/conversion/tmpY4uHZXlocalhost.localdomain@lvm /dev/mapper/cinder--volumes2-volume--181eb2b7--599c--48f1--b589--bf248ab2da71" returned: 0 in 2.981s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:11:11.406 30120 DEBUG oslo_concurrency.processutils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpY4uHZXlocalhost.localdomain@lvm execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:11:12.360 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.335s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:11:12.363 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:11:12.797 30120 DEBUG oslo_concurrency.processutils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpY4uHZXlocalhost.localdomain@lvm" returned: 0 in 1.391s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:11:12.799 30120 DEBUG cinder.image.image_utils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Image conversion details: src /var/lib/cinder/conversion/tmpY4uHZXlocalhost.localdomain@lvm, size 44.00 MB, duration 2.98 sec, destination /dev/mapper/cinder--volumes2-volume--181eb2b7--599c--48f1--b589--bf248ab2da71 _convert_image /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:190
2018-09-14 01:11:12.800 30120 INFO cinder.image.image_utils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Converted 44.00 MB image at 14.74 MB/s
2018-09-14 01:11:12.803 30120 DEBUG cinder.volume.flows.manager.create_volume [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Downloaded image 0bf12814-cbac-4f29-8b63-7c96c2f8b618 ((None, None)) to volume 181eb2b7-599c-48f1-b589-bf248ab2da71 successfully. _copy_image_to_volume /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:582
2018-09-14 01:11:12.810 30120 DEBUG cinder.image.image_utils [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Temporary image 0bf12814-cbac-4f29-8b63-7c96c2f8b618 for user 923cc1ab932046e9bf77dcd7ee5191ad is deleted. fetch /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:676
2018-09-14 01:11:12.812 30120 DEBUG cinder.volume.flows.manager.create_volume [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Marking volume 181eb2b7-599c-48f1-b589-bf248ab2da71 as bootable. _enable_bootable_flag /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:479
2018-09-14 01:11:12.846 30120 DEBUG cinder.volume.flows.manager.create_volume [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Copying metadata from image 0bf12814-cbac-4f29-8b63-7c96c2f8b618 to 181eb2b7-599c-48f1-b589-bf248ab2da71. _handle_bootable_volume_glance_meta /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:437
2018-09-14 01:11:12.847 30120 DEBUG cinder.volume.flows.manager.create_volume [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Creating volume glance metadata for volume 181eb2b7-599c-48f1-b589-bf248ab2da71 backed by image 0bf12814-cbac-4f29-8b63-7c96c2f8b618 with: {'container_format': u'bare', 'min_ram': 0, 'disk_format': u'qcow2', 'image_name': u'tempest-cirros-0.4.0-x86_64-1536901511', 'image_id': u'0bf12814-cbac-4f29-8b63-7c96c2f8b618', 'checksum': u'443b7623e27ecf03dc9e01ee93f67afe', 'min_disk': 0, 'size': 12716032}. _capture_volume_image_metadata /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:616
2018-09-14 01:11:13.030 30120 DEBUG cinder.volume.manager [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Task 'cinder.volume.flows.manager.create_volume.CreateVolumeFromSpecTask;volume:create' (c7df6b34-7675-45a8-8653-61bf26c5f6e0) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'None' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-14 01:11:13.034 30120 DEBUG cinder.volume.manager [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Task 'cinder.volume.flows.manager.create_volume.CreateVolumeOnFinishTask;volume:create, create.end' (2084ae97-70dd-4076-8ed4-7fb6a02549fa) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-14 01:11:13.121 30120 INFO cinder.volume.flows.manager.create_volume [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Volume volume-181eb2b7-599c-48f1-b589-bf248ab2da71 (181eb2b7-599c-48f1-b589-bf248ab2da71): created successfully
2018-09-14 01:11:13.126 30120 DEBUG cinder.volume.manager [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Task 'cinder.volume.flows.manager.create_volume.CreateVolumeOnFinishTask;volume:create, create.end' (2084ae97-70dd-4076-8ed4-7fb6a02549fa) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'None' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-14 01:11:13.131 30120 DEBUG cinder.volume.manager [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Flow 'volume_create_manager' (ae1413d3-c7f5-4fb7-b597-30d4f5bcfbc3) transitioned into state 'SUCCESS' from state 'RUNNING' _flow_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:145
2018-09-14 01:11:13.133 30120 INFO cinder.volume.manager [req-52347554-7e1b-4bfb-a2c1-ddfa8d2d0ac7 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Created volume successfully.
2018-09-14 01:11:13.360 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 0.997s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:11:13.362 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:11:14.382 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.020s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:11:53.844 30120 DEBUG oslo_concurrency.processutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf lvchange -a y --yes -K cinder-volumes2/volume-181eb2b7-599c-48f1-b589-bf248ab2da71 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:11:55.262 30120 DEBUG oslo_concurrency.processutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf lvchange -a y --yes -K cinder-volumes2/volume-181eb2b7-599c-48f1-b589-bf248ab2da71" returned: 0 in 1.418s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:11:55.384 30120 INFO cinder.volume.targets.lio [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Creating iscsi_target for volume: volume-181eb2b7-599c-48f1-b589-bf248ab2da71
2018-09-14 01:11:55.388 30120 DEBUG oslo_concurrency.lockutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 01:11:55.391 30120 DEBUG oslo_concurrency.processutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool create /dev/cinder-volumes2/volume-181eb2b7-599c-48f1-b589-bf248ab2da71 iqn.2010-10.org.openstack:volume-181eb2b7-599c-48f1-b589-bf248ab2da71 Wt3eJkJiUw3jb3SRgNAk VWS4o4brKf8hzJrv False -p3260 -a10.0.20.25 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:11:57.941 30120 DEBUG oslo_concurrency.processutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool create /dev/cinder-volumes2/volume-181eb2b7-599c-48f1-b589-bf248ab2da71 iqn.2010-10.org.openstack:volume-181eb2b7-599c-48f1-b589-bf248ab2da71 Wt3eJkJiUw3jb3SRgNAk VWS4o4brKf8hzJrv False -p3260 -a10.0.20.25" returned: 0 in 2.551s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:11:57.943 30120 DEBUG oslo_concurrency.lockutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.555s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 01:11:57.944 30120 DEBUG oslo_concurrency.lockutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 01:11:57.946 30120 DEBUG oslo_concurrency.processutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:12:00.371 30120 DEBUG oslo_concurrency.processutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets" returned: 0 in 2.425s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:12:00.373 30120 DEBUG oslo_concurrency.lockutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.429s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 01:12:00.375 30120 DEBUG oslo_concurrency.lockutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 01:12:00.377 30120 DEBUG oslo_concurrency.processutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:12:02.885 30120 DEBUG oslo_concurrency.processutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save" returned: 0 in 2.509s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:12:02.889 30120 DEBUG oslo_concurrency.lockutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.514s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 01:12:02.891 30120 DEBUG cinder.volume.targets.iscsi [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Set provider_location to: 10.0.20.25:3260,iqn.2010-10.org.openstack:volume-181eb2b7-599c-48f1-b589-bf248ab2da71 iqn.2010-10.org.openstack:volume-181eb2b7-599c-48f1-b589-bf248ab2da71 0 create_export /usr/lib/python2.7/site-packages/cinder/volume/targets/iscsi.py:215
2018-09-14 01:12:02.915 30120 DEBUG oslo_concurrency.lockutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 01:12:02.919 30120 DEBUG oslo_concurrency.processutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool add-initiator iqn.2010-10.org.openstack:volume-181eb2b7-599c-48f1-b589-bf248ab2da71 Wt3eJkJiUw3jb3SRgNAk VWS4o4brKf8hzJrv iqn.1994-05.com.redhat:f393eeb713 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:12:05.753 30120 DEBUG oslo_concurrency.processutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool add-initiator iqn.2010-10.org.openstack:volume-181eb2b7-599c-48f1-b589-bf248ab2da71 Wt3eJkJiUw3jb3SRgNAk VWS4o4brKf8hzJrv iqn.1994-05.com.redhat:f393eeb713" returned: 0 in 2.835s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:12:05.756 30120 DEBUG oslo_concurrency.lockutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.840s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 01:12:05.757 30120 DEBUG oslo_concurrency.lockutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 01:12:05.759 30120 DEBUG oslo_concurrency.processutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:12:08.588 30120 DEBUG oslo_concurrency.processutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save" returned: 0 in 2.829s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:12:08.590 30120 DEBUG oslo_concurrency.lockutils [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.833s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 01:12:08.640 30120 INFO cinder.volume.manager [req-682b4034-0701-49d3-96fa-e2726ab61f00 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Initialize volume connection completed successfully.
2018-09-14 01:12:09.535 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:12:09.536 30120 DEBUG cinder.manager [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-14 01:12:09.591 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:12:09.591 30120 DEBUG cinder.volume.drivers.lvm [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-14 01:12:09.593 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:12:10.856 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.263s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:12:10.859 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:12:11.070 30120 DEBUG cinder.coordination [req-44a5df97-8540-417c-bd76-1de212ee0422 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Lock "/var/lib/cinder/cinder-181eb2b7-599c-48f1-b589-bf248ab2da71" acquired by "attach_volume" :: waited 0.001s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:175
2018-09-14 01:12:11.315 30120 INFO cinder.volume.manager [req-44a5df97-8540-417c-bd76-1de212ee0422 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Attaching volume 181eb2b7-599c-48f1-b589-bf248ab2da71 to instance 6cc38e3c-8c37-44c3-a3b2-8aafeca153bc at mountpoint /dev/vdb on host None.
2018-09-14 01:12:11.727 30120 INFO cinder.volume.manager [req-44a5df97-8540-417c-bd76-1de212ee0422 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Attach volume completed successfully.
2018-09-14 01:12:11.728 30120 DEBUG cinder.coordination [req-44a5df97-8540-417c-bd76-1de212ee0422 923cc1ab932046e9bf77dcd7ee5191ad f4b8aa14ab144c71ad8c02aaabb0b4f4 - 897c9720461f4c5eac6812853d39b8d2 897c9720461f4c5eac6812853d39b8d2] Lock "/var/lib/cinder/cinder-181eb2b7-599c-48f1-b589-bf248ab2da71" released by "attach_volume" :: held 0.658s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:187
2018-09-14 01:12:11.983 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.124s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:12:11.986 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:12:12.887 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 0.902s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:12:12.890 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:12:13.916 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.026s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:13:09.532 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:13:09.535 30120 DEBUG cinder.manager [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-14 01:13:09.593 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:13:09.596 30120 DEBUG cinder.volume.drivers.lvm [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-14 01:13:09.597 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:13:10.820 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.223s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:13:10.824 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:13:11.977 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.153s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:13:11.979 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:13:13.004 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.025s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:13:13.007 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:13:13.997 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.990s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:14:09.531 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:14:09.535 30120 DEBUG cinder.manager [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-14 01:14:09.607 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:14:09.609 30120 DEBUG cinder.volume.drivers.lvm [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-14 01:14:09.611 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:14:10.993 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.382s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:14:10.997 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:14:12.362 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.366s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:14:12.366 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:14:13.697 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.332s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:14:13.701 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:14:15.038 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.337s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:14:54.208 30120 DEBUG oslo_concurrency.lockutils [req-6d387360-09db-4f25-9a18-8fb5df47980c c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 01:14:54.212 30120 DEBUG oslo_concurrency.processutils [req-6d387360-09db-4f25-9a18-8fb5df47980c c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool delete-initiator iqn.2010-10.org.openstack:volume-181eb2b7-599c-48f1-b589-bf248ab2da71 iqn.1994-05.com.redhat:f393eeb713 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:14:56.852 30120 DEBUG oslo_concurrency.processutils [req-6d387360-09db-4f25-9a18-8fb5df47980c c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool delete-initiator iqn.2010-10.org.openstack:volume-181eb2b7-599c-48f1-b589-bf248ab2da71 iqn.1994-05.com.redhat:f393eeb713" returned: 0 in 2.641s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:14:56.855 30120 DEBUG oslo_concurrency.lockutils [req-6d387360-09db-4f25-9a18-8fb5df47980c c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.647s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 01:14:56.859 30120 DEBUG oslo_concurrency.lockutils [req-6d387360-09db-4f25-9a18-8fb5df47980c c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 01:14:56.861 30120 DEBUG oslo_concurrency.processutils [req-6d387360-09db-4f25-9a18-8fb5df47980c c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:14:59.389 30120 DEBUG oslo_concurrency.processutils [req-6d387360-09db-4f25-9a18-8fb5df47980c c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save" returned: 0 in 2.528s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:14:59.393 30120 DEBUG oslo_concurrency.lockutils [req-6d387360-09db-4f25-9a18-8fb5df47980c c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.534s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 01:14:59.395 30120 INFO cinder.volume.manager [req-6d387360-09db-4f25-9a18-8fb5df47980c c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Terminate volume connection completed successfully.
2018-09-14 01:14:59.784 30120 DEBUG cinder.coordination [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "/var/lib/cinder/cinder-181eb2b7-599c-48f1-b589-bf248ab2da71-detach_volume" acquired by "detach_volume" :: waited 0.001s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:175
2018-09-14 01:14:59.882 30120 INFO cinder.volume.manager [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Detaching volume 181eb2b7-599c-48f1-b589-bf248ab2da71 from instance 6cc38e3c-8c37-44c3-a3b2-8aafeca153bc.
2018-09-14 01:14:59.885 30120 DEBUG oslo_concurrency.lockutils [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 01:14:59.886 30120 DEBUG oslo_concurrency.processutils [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:15:02.054 30120 DEBUG oslo_concurrency.processutils [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets" returned: 0 in 2.168s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:15:02.057 30120 DEBUG oslo_concurrency.lockutils [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.172s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 01:15:02.059 30120 INFO cinder.volume.targets.lio [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Removing iscsi_target: 181eb2b7-599c-48f1-b589-bf248ab2da71
2018-09-14 01:15:02.060 30120 DEBUG oslo_concurrency.lockutils [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 01:15:02.063 30120 DEBUG oslo_concurrency.processutils [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool delete iqn.2010-10.org.openstack:volume-181eb2b7-599c-48f1-b589-bf248ab2da71 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:15:04.578 30120 DEBUG oslo_concurrency.processutils [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool delete iqn.2010-10.org.openstack:volume-181eb2b7-599c-48f1-b589-bf248ab2da71" returned: 0 in 2.516s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:15:04.580 30120 DEBUG oslo_concurrency.lockutils [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.519s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 01:15:04.581 30120 DEBUG oslo_concurrency.lockutils [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 01:15:04.583 30120 DEBUG oslo_concurrency.processutils [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:15:07.362 30120 DEBUG oslo_concurrency.processutils [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save" returned: 0 in 2.779s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:15:07.365 30120 DEBUG oslo_concurrency.lockutils [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.783s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 01:15:07.651 30120 INFO cinder.volume.manager [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Detach volume completed successfully.
2018-09-14 01:15:07.653 30120 DEBUG cinder.coordination [req-55a49a03-d2d3-43d2-92ff-37222bdb60b7 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "/var/lib/cinder/cinder-181eb2b7-599c-48f1-b589-bf248ab2da71-detach_volume" released by "detach_volume" :: held 7.869s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:187
2018-09-14 01:15:09.536 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:15:09.538 30120 DEBUG cinder.manager [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-14 01:15:09.601 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:15:09.602 30120 DEBUG cinder.volume.drivers.lvm [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-14 01:15:09.605 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:15:10.787 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.183s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:15:10.790 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:15:11.720 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.930s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:15:11.725 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:15:12.649 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 0.924s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:15:12.651 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:15:13.744 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.093s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:15:18.957 30120 DEBUG cinder.coordination [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "/var/lib/cinder/cinder-181eb2b7-599c-48f1-b589-bf248ab2da71-delete_volume" acquired by "delete_volume" :: waited 0.001s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:175
2018-09-14 01:15:19.141 30120 DEBUG oslo_concurrency.lockutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 01:15:19.143 30120 DEBUG oslo_concurrency.processutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:15:21.970 30120 DEBUG oslo_concurrency.processutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets" returned: 0 in 2.827s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:15:21.973 30120 DEBUG oslo_concurrency.lockutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.833s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 01:15:21.975 30120 INFO cinder.volume.targets.iscsi [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Skipping remove_export. No iscsi_target is presently exported for volume: 181eb2b7-599c-48f1-b589-bf248ab2da71
2018-09-14 01:15:21.979 30120 DEBUG oslo_concurrency.processutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2/volume-181eb2b7-599c-48f1-b589-bf248ab2da71 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:15:23.389 30120 DEBUG oslo_concurrency.processutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2/volume-181eb2b7-599c-48f1-b589-bf248ab2da71" returned: 0 in 1.411s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:15:23.393 30120 DEBUG oslo_concurrency.processutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvdisplay --noheading -C -o Attr cinder-volumes2/volume-181eb2b7-599c-48f1-b589-bf248ab2da71 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:15:24.791 30120 DEBUG oslo_concurrency.processutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvdisplay --noheading -C -o Attr cinder-volumes2/volume-181eb2b7-599c-48f1-b589-bf248ab2da71" returned: 0 in 1.398s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:15:24.797 30120 DEBUG oslo_concurrency.processutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf lvremove --config activation { retry_deactivation = 1}  -f cinder-volumes2/volume-181eb2b7-599c-48f1-b589-bf248ab2da71 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:15:26.435 30120 DEBUG oslo_concurrency.processutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf lvremove --config activation { retry_deactivation = 1}  -f cinder-volumes2/volume-181eb2b7-599c-48f1-b589-bf248ab2da71" returned: 0 in 1.639s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:15:26.439 30120 INFO cinder.volume.drivers.lvm [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Successfully deleted volume: 181eb2b7-599c-48f1-b589-bf248ab2da71
2018-09-14 01:15:26.570 30120 DEBUG cinder.quota [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Created reservations ['0c57d609-272b-482e-9464-82d5bb4991de', '2387449f-0436-4493-9416-33377b0e162c', 'fbc5d745-e72e-454c-8343-27f113fef8aa', 'ecb1d4a0-c485-4bf7-9b45-9056fa792e25'] reserve /usr/lib/python2.7/site-packages/cinder/quota.py:1029
2018-09-14 01:15:26.675 30120 DEBUG cinder.volume.utils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Can not find volume 181eb2b7-599c-48f1-b589-bf248ab2da71 at notify usage _usage_from_volume /usr/lib/python2.7/site-packages/cinder/volume/utils.py:99
2018-09-14 01:15:26.808 30120 DEBUG cinder.volume.drivers.lvm [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-14 01:15:26.810 30120 DEBUG oslo_concurrency.processutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:15:28.152 30120 DEBUG oslo_concurrency.processutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.342s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:15:28.156 30120 DEBUG oslo_concurrency.processutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:15:29.668 30120 DEBUG oslo_concurrency.processutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.512s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:15:29.672 30120 DEBUG oslo_concurrency.processutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:15:31.142 30120 DEBUG oslo_concurrency.processutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.470s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:15:31.147 30120 DEBUG oslo_concurrency.processutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:15:32.431 30120 DEBUG oslo_concurrency.processutils [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.284s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:15:32.437 30120 DEBUG cinder.manager [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-14 01:15:32.691 30120 INFO cinder.volume.manager [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Deleted volume successfully.
2018-09-14 01:15:32.716 30120 DEBUG cinder.coordination [req-71b2524f-56ec-4a45-a11e-87502e42eec0 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "/var/lib/cinder/cinder-181eb2b7-599c-48f1-b589-bf248ab2da71-delete_volume" released by "delete_volume" :: held 13.759s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:187
2018-09-14 01:16:09.539 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:16:09.543 30120 DEBUG cinder.manager [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-14 01:16:09.608 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:16:09.614 30120 DEBUG cinder.volume.drivers.lvm [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-14 01:16:09.616 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:16:11.120 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.504s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:16:11.123 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:16:12.425 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.302s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:16:12.430 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:16:13.770 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.341s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:16:13.774 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:16:15.104 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.331s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:17:09.538 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:17:09.541 30120 DEBUG cinder.manager [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-14 01:17:09.595 30120 DEBUG oslo_service.periodic_task [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-14 01:17:09.596 30120 DEBUG cinder.volume.drivers.lvm [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-14 01:17:09.598 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:17:10.864 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.266s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:17:10.867 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:17:11.967 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.101s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:17:11.971 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:17:13.048 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.077s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-14 01:17:13.053 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-14 01:17:14.108 30120 DEBUG oslo_concurrency.processutils [req-a108b6cc-1e94-4d99-bdbd-017d9e95353d - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.055s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
